scss/civicrm/administer/contribute/_managepremiums.scss

```diff
@@ -1,7 +1,29 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
 &#{civi-page('admin-contribute-managepremiums')} {
   #crm-main-content-wrapper {
     .dataTables_wrapper {
       box-shadow: none;
     }
   }
+
+  #{civi-dialog()} {
+    .CRM_Contribute_Form_ManagePremiums {
+      .crm-collapsible {
+        margin: 0 -20px;
+
+        .collapsible-title {
+          @include accordion-header;
+          font-size: $font-size-base;
+          margin-bottom: 15px;
+        }
+      }
+
+      .premium-full-disabled {
+        input[type='button'] {
+          margin: 20px 0;
```

Comment:
why was there a problem with vertical margin in the first place? was the problem there also in the original style?

Reply:
no, there was no problem with vanilla civicrm,
Buttons do not have any margins around it.
It's just for a button between content (See dark gray contribute button )
![image](https://user-images.githubusercontent.com/26058635/37083093-7163ba5a-2214-11e8-8d62-e93346532a43.png)

Reply:
so is it a problem that we introduced ourselves? why?

Reply:
Nope,
![image](https://user-images.githubusercontent.com/26058635/37084165-c35c277c-2217-11e8-9ebe-934f13cf887d.png)
It's by default like that, we are adding margins to make it look more cleaner.